st2actions/st2actions/container/base.py

```diff
@@ -116,7 +119,8 @@ def _do_run(self, runner, runnertype_db, action_db, liveaction_db):
                                                               action_parameters=action_params)
             extra = {'runner': runner, 'parameters': resolved_action_params}
             LOG.debug('Performing run for runner: %s' % (runner.runner_id), extra=extra)
-            (status, result, context) = runner.run(action_params)
+            with CounterWithTimer(key=format_metrics_key(action_db, key='action')):
```

Comment:
It would be good to use keyword arguments when passing action_db to format_metrics_key function for consistency reasons.

Reply:
And it might even make sense to create another decorator class which inherits from `CounterWithTimer` called `CounterWithTimerForActionDB` and `CounterWithTimerForLiveActionDB` or similar which takes `action_db` directly and generates key for it (this way we don't need to call `format_metrics_key` here since that would happen inside the decorator).